src/main/java/org/fogbowcloud/arrebol/ArrebolController.java

```diff
@@ -158,4 +192,41 @@ public TaskState getTaskState(String taskId) {
         //FIXME:
         return null;
     }
+
+    private void setWorkerCreator(String type) throws IOException {
+        switch (type) {
+            case DockerConstants.DOCKER_TYPE:
+                this.workerCreator = new DockerWorkerCreator();
+                break;
+            case RawConstants.RAW_TYPE:
+                this.workerCreator = new RawWorkerCreator();
+                break;
+        }
+    }
+
+    private void updateJobState(Job job) {
```

Comment:
drop a comment here indicating that arrebol does not change job state internally, so we need this workaround

Reply:
Done.